_overviews/scala3-book/taste-control-structures.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Control Structures
+type: section
+description: This section demonstrates Scala 3 control structures.
+num: 8
+previous-page: taste-vars-data-types
+next-page: taste-modeling
+---
+
+
+Scala has the programming language control structures you find in other languages, and also has powerful `for` expressions and `match` expressions:
+
+- `if`/`else`
+- `for` loops and expressions
+- `match` expressions
+- `while` loops
+- `try`/`catch`
+
+These structures are demonstrated in the following examples.
+
+
+
+## `if`/`else`
+
+Scala’s `if`/`else` control structure is similar to other languages:
```

Comment:
It looks similar but it is not because, as shown just after, its an expression and not a statement.

```suggestion
Scala’s `if`/`else` control structure looks similar to other languages:
```